src/routing/tests/mod.rs

```diff
@@ -1174,7 +1226,7 @@ fn create_relocation_trigger(sk: &bls::SecretKey, age: u8) -> Result<(Vote, Proo
 
         let signature = sk.sign(&bincode::serialize(&vote.as_signable())?);
 
-        if relocation::check(age, &signature) {
+        if relocation::check(age, &signature) && !relocation::check(age + 1, &signature) {
```

Comment:
why the extra check?

Reply:
Because we only want to trigger relocation of nodes with that exact age, not with any higher age.